### PR TITLE
Change import UI to expect import to happen out-of-band

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -24,7 +24,9 @@
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "babel-root-import": "^3.2.2",
```
Comment (justinvdm) [inline client/package.json]: It seems I forgot to add a dependency earlier, prevented building properly when I did a clean install for the client side. Sorry for adding this irrelevant change to this PR.

**Comment by justinvdm** [comment]
@miltontony ready for review

**Comment by justinvdm** [comment]
Checking now displays errors again. Ready for re-review.

**Comment by miltontony** [comment]
:+1: :cake: thanks!

**Pull request merged**